class/Command/SaveInternship.php

```diff
@@ -383,6 +383,9 @@ public function execute()
             throw $e;
         }
 
+        //Commit to save changes in case of workflow error
+        \PHPWS_DB::commit();
+
```

Comment:
We should probably open a new transaction here (`\PHPWS_DB::begin()`), before we do the workflow stuff. There's another `COMMIT` on [line 421](https://github.com/caldwellc1/InternshipInventory/blob/8382f1a64b74ba04d46e770c77f2f69194c9aa6e/class/Command/SaveInternship.php#L421), but there won't be any open transaction at that time unless we open a new one here.